src/os/bluestore/BlueStore.h

```diff
@@ -208,28 +212,28 @@ class BlueStore : public ObjectStore,
       assert(writing.empty());
     }
 
-    void _add_buffer(Buffer *b) {
-      cache->_audit_lru("_add_buffer start");
+    void _add_buffer(Buffer *b, int level, Buffer *near) {
```

Comment:
Why not defining such methods really protected? I suppose this is pretty doable.